lib/actions/grants/refresh_token.js

```diff
@@ -71,7 +75,13 @@ module.exports.handler = function getRefreshTokenHandler(provider) {
     if (refreshTokenRotation === 'rotateAndConsume') {
       try {
         if (refreshToken.consumed) {
-          throw new InvalidGrant('refresh token already used');
+          // if we are consumed check if we are in the grace period
+          gracePeriod = conf.refreshTokenGracePeriod[refreshToken.clientId] || conf.refreshTokenGracePeriodDefault
```

Comment:
Please add a `let` or `const` if you are declaring a variable. Otherwise, it becomes a global variable.